app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 hummer_level: int,
+                 favourite_dish: str) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return self._hummer_level + 4
+
+    def player_info(self) -> str:
+        return (f"Dwarf warrior {self.nickname}."
+                f" {self.nickname} has a hummer of the"
+                f" {self._hummer_level} level"
+                )

```

Comment:
Code Style: Consider using a more descriptive method name than 'player_info'. Something like 'get_player_info' would be more clear and consistent with the 'get_rating' method.